app/controllers/application_controller.rb

```diff
@@ -1,12 +1,24 @@
 class ApplicationController < ActionController::Base
+  include Pundit
   before_action :authenticate_user!
 
   before_action :configure_permitted_parameters, if: :devise_controller?
 
+  # after_action :verify_authorized, unless: :devise_controller?
+  # after_action :verify_policy_scoped, only: :index, unless: :devise_controller?
+
```

Comment:
You do want these not commented out. It's our safeguard to make sure we have policies set up